ESRGAN/readme.md

```diff
@@ -0,0 +1,5 @@
+ * You have to install `ESRGAN` models in this folder for upscaling.
```

Comment:
```suggestion
 * Place `ESRGAN` models in this folder for upscaling.
```

Reply:
Actually, it looks like this file should probably move to `models/ESRGAN` now.

Reply:
Fixed in https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/1375/commits/e35521f36684498cccd6aec4bcee50b8156fc989